src/components/common/Badge/EmojiBadge.jsx

```diff
@@ -0,0 +1,37 @@
+//이모지 뱃지 컴포넌트
+
+import { cva } from 'class-variance-authority';
+import PropTypes from 'prop-types';
+import { cn } from '@/utils/style';
+
+const emojiBadgeStyle = cva(
+  'inline-flex items-center justify-center rounded-full bg-black/50 text-white font-16-regular',
+  {
+    variants: {
+      size: {
+        md: 'px-3 py-2 h-[36px] text-sm',
+        lg: 'px-4 py-2 h-[40px] text-base',
+      },
+    },
+    defaultVariants: {
+      size: 'md',
+    },
+  }
+);
+
+const EmojiBadge = ({ emoji, count, size }) => {
+  return (
+    <span className={cn(emojiBadgeStyle({ size }))}>
+      <span className="mr-1">{emoji}</span>
+      {count !== undefined && <span>{count}</span>}
+    </span>
+  );
+};
+
+EmojiBadge.propTypes = {
+  emoji: PropTypes.string.isRequired,
+  count: PropTypes.number,
+  size: PropTypes.oneOf(['md', 'lg']),
+};
+
+export default EmojiBadge;
```

Comment:
이모지 뱃지 부분은 [아까 리뷰드렸던 부분](https://github.com/Sprint-19-Rolling/Rolling-Frontend/pull/28#discussion_r2383961459)이 반영되지 않은 것 같습니다....🥲
피그마 시안이랑 다른 부분이 있어서 그 부분 반영 부탁드립니다..!!!
> 요렇게 s 사이즈를 추가해야할 것 같아요! 

```suggestion
import { cva } from 'class-variance-authority';
import PropTypes from 'prop-types';
import { cn } from '@/utils/style';

const emojiBadgeStyle = cva(
  'inline-flex items-center justify-center rounded-full bg-black/50 text-white font-16-regular',
  {
    variants: {
      size: {
        s: 'px-3 py-2 h-[28px] font-14-regular',
        md: 'px-3 py-2 h-[36px]',
        lg: 'px-4 py-2 h-[38px]',
      },
    },
  }
);

const EmojiBadge = ({ emoji, count, size = 'md' }) => {
  return (
    <span className={cn(emojiBadgeStyle({ size }))}>
      <span className="mr-1.5">{emoji}</span>
      {count !== undefined && <span>{count}</span>}
    </span>
  );
};

EmojiBadge.propTypes = {
  emoji: PropTypes.string.isRequired,
  count: PropTypes.number,
  size: PropTypes.oneOf(['s', 'md', 'lg']),
};

export default EmojiBadge;
```

그리고 Badge(폴더명)을 폴더명 규칙에 맞게 badge로 변경 부탁드립니다!
폴더명 규칙은 소문자 케밥케이스입니다~~!